docs/LogGuide.md

```diff
@@ -2767,3 +2771,50 @@ Parsed Log Line Examples:
 ```
 
 <!-- AUTO-GENERATED-CONTENT:END (logLines:type=AbilityExtra&lang=en-US) -->
+
+<a name="line265"></a>
+
+### Line 265 (0x109): ContentFinderSettings
+
+This log line tracks the current Content Finder settings.
+`inContentFinderContent` is whether the current zone supports Content Finder settings.
+
+`unrestrictedParty`, `minimalItemLevel`, `silenceEcho`, `explorerMode`, and `levelSync` are `0` for disabled or `1` for enabled.
```

Comment:
It may be worth noting here that the `0` and `1` values are directly from the game client, so they could change in the future. Something like this maybe? (that's why they're not coerced into boolean `True`/`False`)

> Values for `...`, `...` are pulled directly from the game. As of patch 6.5.1, a value of `0` indicates that the setting is disabled, and a value of `1` indicates that it is enabled.

Reply:
Sounds good, I'll make the change.